boxwise_flask/models.py

```diff
@@ -1,18 +1,53 @@
 """Model definitions for database"""
-from peewee import CharField, CompositeKey, DateField, DateTimeField
+from peewee import CharField, CompositeKey, DateField, DateTimeField, IntegerField
 from playhouse.shortcuts import model_to_dict
 
 from .app import db
 
 
-class Person(db.Model):
-    firstname = CharField()
-    lastname = CharField()
-    camp_id = CharField()
-    id = CharField()
+class Stock(db.Model):
+    # INSERT INTO stock (
```

Comment:
I'd probably remove the SQL. Unless it has some other use then potentially annotate it.